torch/csrc/jit/passes/symbolic_shape_analysis.cpp

```diff
@@ -135,6 +135,25 @@ bool symbolicShapeAnalysisTestModeEnabled() {
   return symbolic_shape_analysis_test_mode;
 }
 
+namespace {
+
+bool isListOfInts(const TypePtr& type) {
+  return type->cast<ListType>() &&
+      type->cast<ListType>()->getElementType()->cast<IntType>();
+}
+
+TupleTypePtr mapTensorTupleType(
```

Comment:
Not sure what this function does from the name, do you mind adding a comment explaining it?